firmware/common/ui_widget.cpp

```diff
@@ -1327,10 +1327,11 @@ void NewButton::paint(Painter& painter) {
         style.background);
 
     int y = r.top();
+
     if (bitmap_) {
         int offset_y = vertical_center_ ? (r.height() / 2) - (bitmap_->size.height() / 2) : 6;
         Point bmp_pos = {r.left() + (r.width() / 2) - (bitmap_->size.width() / 2), r.top() + offset_y};
-        y += bitmap_->size.height() - offset_y;
+        y += bitmap_->size.height() + offset_y;
```

Comment:
Please explain why you changed the sign ?

Reply:
That's to satisfy the new change of text drawing policy. 

for the btngrid(tile) the vertical_center_ default false so offset_y fixed 6

In previous code, it draws the bitmap, and kept the y value, and draw the text with a slight offset (AKA y padding). 

In now code I need the text to draw on top to allow two lines, thus y doesn't kept following the value after draw the bitmap.

As for why it from - to + i think it was a coincidence because i did a lot of change to make it looks like same as original, and accidentally used the 6 too as padding at the end but added instead of minus.

Reply:
But since you mentioned, which made me thinking if it is worth to use this expensive change just for implementing draw two lines. The space complexity should be same but maybe still to expensive. Can't see rendering time cost difference tho